lblod/ember-rdfa-editor
# GN-4538: Allow to remove node via button with confirmation modal

### Overview

Allows to remove node via button with confirmation dialog

##### connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4583


### Setup

1. Checkout
2. `npm run start` 

### How to test/reproduce

1. Go to "Editable Nodes POC"
2. Insert RdfaTest
3. See below

https://github.com/lblod/ember-rdfa-editor/assets/769698/032ed7c6-ef85-4eff-8c36-90cd98c7c70a

### Challenges/uncertainties

1. I'm not sure it needs a confirmation dialog at all. We're not asking user to confirm when he's deleting with backspace.
2. Changed "Convert to YYY node" button to be blue instead of red, so the "delete" button stands out
3. Figuring out best way to remove node by just pressing backspace/delete, right now when clicking on a node, caret appears at the closest text instead of "selecting" the node, so pressing backspace/delete naturally starts deleting the text.
  * When backspacing into a node we can adopt `selectBlockRdfaNode`
  * When selecting the node - ???


### Checks PR readiness
